### PR TITLE
DISTMYSQL-23 Documentation for 8.0.19 release

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: source/installing.rst [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+   .. code-block:: bash
+  
+      $ sudo yum install percona-server
+
+#. Install the components. Use the commands below to install the required components: 
+   
+   .. code-block:: bash
+   
+      $ #Install Orchestrator
+      $ sudo yum install percona-orchestrator
```
Comment (EvgeniyPatlan) [inline source/installing.rst]: need to add note that epel-release package should be installed as 2 orchestrator dependencies (jq and oniguruma) should be installed from it

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: source/installing.rst (hunk 1 above)
Comment (nastena1606) [reply source/installing.rst]: Is it appllicable for all platforms or for Centos 7, like in Postgresql docs?

Comment (EvgeniyPatlan) [reply source/installing.rst]: Yes it is needed only on centos7. On centos8 packages would be installed from default AppStream repo.

**Review by EvgeniyPatlan — APPROVED** [review]

File: source/installing.rst [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+   Make sure to run all commands as root or via |sudo|.
+
+Install |PS|-based deployment on Debian / Ubuntu
+--------------------------------------------------------------------------------
+
+1. Install |PS|:
+   
+   .. code-block:: bash
+  
+      $ sudo apt-get install percona-server
```
Comment (ghost) [inline source/installing.rst]: I think also we need to install client
percona-server-client
But not sure, maybe percona-server-client depends on percona-server. 
And we have additional packages:
      - percona-server-test
      - percona-server-dbg
      - percona-server-source
      - percona-server-tokudb
      - percona-server-rocksdb

File: source/installing.rst [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+#. Install the components. Use the commands below to install the required components: 
+   
+   .. code-block:: bash
+   
+      $ #Install Percona XtraBackup
+      $ sudo apt-get install percona-xtrabackup-80
+      $ #Install Percona Toolkit
+      $ sudo apt-get install percona-toolkit
+      $ #Install Orchestrator
+      $ sudo apt-get install percona-orchestrator
```
Comment (ghost) [inline source/installing.rst]: For orchestrator we have a 3 packages:
        - percona-orchestrator-cli
        - percona-orchestrator-client
        - percona-orchestrator

File: source/installing.rst (hunk 2 above)
Comment (nastena1606) [reply source/installing.rst]: I guess it does https://repo.percona.com/pdps-8.0.19/apt/pool/testing/p/percona-server/

File: source/installing.rst (hunk 3 above)
Comment (nastena1606) [reply source/installing.rst]: Do we need to explicitly mention all three?

File: source/installing.rst [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+   .. code-block:: bash
+   
+      $ sudo apt-get install percona-xtradb-cluster
+
+
+#. Install HAProxy:  
+
+   .. code-block:: bash
+   
+      $ sudo apt-get install percona-haproxy
```
Comment (ghost) [inline source/installing.rst:203]: Full list of packages:
    - percona-haproxy
    - percona-haproxy-doc
    - percona-vim-haproxy
    - percona-haproxy-dbgsym

File: source/installing.rst [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+       * `Percona Toolkit <https://www.percona.com/doc/percona-toolkit/2.2/installation.html>`_
+
+Install |PXC|-based deployment on Debian / Ubuntu
+--------------------------------------------------------------------------------
+
+1. Install |PXC|:
+   
+   .. code-block:: bash
+   
+      $ sudo apt-get install percona-xtradb-cluster
```
Comment (ghost) [inline source/installing.rst:196]: Full list of packages:
    - percona-xtradb-cluster-full
    - percona-xtradb-cluster-client
    - percona-xtradb-cluster-common
    - percona-xtradb-cluster-dbg
    - percona-xtradb-cluster-garbd-debug
    - percona-xtradb-cluster-garbd
    - percona-xtradb-cluster-server
    - percona-xtradb-cluster-server-debug
    - percona-xtradb-cluster-test
    - percona-xtradb-cluster

I think we need to install full or cluster+client at least.

File: source/installing.rst [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+       * `Percona Toolkit <https://www.percona.com/doc/percona-toolkit/2.2/installation.html>`_
+ 
+Install |PXC|-based deployment on |RHEL| / |centos|
+--------------------------------------------------------------------------------
+
+1. Install |PXC|:
+   
+   .. code-block:: bash
+   
+      $ sudo yum install percona-xtradb-cluster
```
Comment (ghost) [inline source/installing.rst:228]: List of packages:
    - percona-xtradb-cluster-full
    - percona-xtradb-cluster
    - percona-xtradb-cluster-client
    - percona-xtradb-cluster-debuginfo
    - percona-xtradb-cluster-devel
    - percona-xtradb-cluster-garbd
    - percona-xtradb-cluster-server
    - percona-xtradb-cluster-shared
    - percona-xtradb-cluster-shared-compat
    - percona-xtradb-cluster-test

File: source/installing.rst (hunk 3 above)
Comment (ghost) [reply source/installing.rst]: Depends on what do you need. I think all this packages needed, but maybe in documentation need to split it on 3 steps of installation process. Not sure how better.

File: source/installing.rst (hunk 2 above)
Comment (ghost) [reply source/installing.rst]: Also we haven't percona-server package, only percona-server-server for server

File: source/installing.rst (hunk 6 above)
Comment (mu-samoylov) [reply source/installing.rst:208]: And we haven't   package with name percona-xtradb-cluster

**Review by borys-belinsky — APPROVED** [review]

File: source/installing.rst (hunk 2 above)
Comment (EvgeniyPatlan) [reply source/installing.rst]: percona-server-client will be installed as the dependency for percona-server-server

Comment (nastena1606) [reply source/installing.rst]: Will the above listed packages have to be installed explicitly?

File: source/installing.rst (hunk 4 above)
Comment (nastena1606) [reply source/installing.rst:203]: Same question, is it enough to install percona-haproxy only to work with it? Or we must mention / install the full list of packages?

**Pull request merged**